### PR TITLE
DO NOT SQUASH Absorb TSFC

### PR DESCRIPTION
**Pull request by connorjward**
Happily the changes here are extremely minor.

Goes with https://github.com/firedrakeproject/fiat/pull/104

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -30,9 +30,7 @@ dependencies = [
   "scipy",
   "sympy",
   "fenics-ufl @ git+https://github.com/firedrakeproject/ufl.git",
-  "fenics-fiat @ git+https://github.com/firedrakeproject/fiat.git",
-  "finat @ git+https://github.com/FInAT/FInAT.git",
-  "tsfc @ git+https://github.com/firedrakeproject/tsfc.git",
+  "fenics-fiat @ git+https://github.com/firedrakeproject/fiat.git@connorjward/add-finat-gem",
```
Comment (connorjward) [inline pyproject.toml:33]:
```suggestion
  "fenics-fiat @ git+https://github.com/firedrakeproject/fiat.git",
```

File: requirements-git.txt [hunk 2]
```diff
@@ -1,7 +1,5 @@
 git+https://github.com/firedrakeproject/ufl.git#egg=fenics-ufl
-git+https://github.com/firedrakeproject/fiat.git#egg=fenics-fiat
-git+https://github.com/FInAT/FInAT.git#egg=finat
-git+https://github.com/firedrakeproject/tsfc.git#egg=tsfc
+git+https://github.com/firedrakeproject/fiat.git@connorjward/add-finat-gem#egg=fenics-fiat
```
Comment (connorjward) [inline requirements-git.txt:2]:
```suggestion
git+https://github.com/firedrakeproject/fiat.git#egg=fenics-fiat
```

File: firedrake/scripts/firedrake-zenodo [hunk 3]
```diff
@@ -16,35 +16,25 @@ import datetime
...
     ("petsc", "Portable, Extensible Toolkit for Scientific Computation"),
-    ("petsc4py", "The Python interface to PETSc"),
     ("loopy", "Transformation-Based Generation of High-Performance CPU/GPU Code"),
-    ("slepc", "Scalable Library for Eigenvalue Problem Computations"),
-    ("slepc4py", "The Python interface to SLEPc")])
+    ("slepc", "Scalable Library for Eigenvalue Problem Computations")])
 
 projects = dict(
     [("firedrake", "firedrakeproject"),
-     ("tsfc", "firedrakeproject"),
```
[the remark this answers is not in the document]
Comment (connorjward) [reply firedrake/scripts/firedrake-zenodo:35]: That is a great point. We could add TSFC to the list of `optional_components` below (and do the same for PyOP2).

[the remark this answers is not in the document]
Comment (connorjward) [reply firedrake/scripts/firedrake-zenodo:35]: Done.

**Comment by ksagiyam** [comment]
Looks good to me. Is it better if we move `src/tsfc` etc to `src/tsfc_old` etc to clarify that we are no longer using them?

**Comment by connorjward** [comment]
> Looks good to me. Is it better if we move `src/tsfc` etc to `src/tsfc_old` etc to clarify that we are no longer using them? (upon `firedrake-update`)

Done.

**Review by ksagiyam — APPROVED** [review]

**Pull request merged**